### PR TITLE
[BPK-3060]: Updated docs to use correct logo on homepage

### PR DESCRIPTION
**Pull request by ojcurtis**
The pride logo has also been corrected too.

Before:
Screenshot 2019-10-07 at 11 50 56

After:
Screenshot 2019-10-07 at 11 51 02

### CODE REVIEW

File: docs/src/helpers/logo-helper.js [hunk 1]
```diff
@@ -17,10 +17,18 @@
  */
 
 import BackpackLogoWhite from '../static/backpack-logo-white.svg';
+import BackpackLogoGrey from '../static/backpack-logo-sky-grey.svg';
```
Comment (georgegillams) [inline docs/src/helpers/logo-helper.js]: while we're at this, can we rename these to `backpack-logo-light` and `backpack-logo-dark`?

File: docs/src/helpers/logo-helper.js [hunk 2]
```diff
@@ -17,10 +17,18 @@
...
 import BackpackLogoPride from '../static/backpack-logo-pride.svg';
+import BackpackLogoPrideGrey from '../static/backpack-logo-pride-sky-grey.svg';
 
-export const getBackpackLogo = () => {
+export const getBackpackLogo = (isHeroImage = false) => {
```
Comment (georgegillams) [inline docs/src/helpers/logo-helper.js]: Can we rename this to be more `why` and less `what`?
```suggestion
export const getBackpackLogo = (lightVariant = false) => {
```

File: docs/src/helpers/logo-helper.js (hunk 1 above)
Comment (ojcurtis) [reply docs/src/helpers/logo-helper.js]: So these are in line with the spec from Will so will need to check as its not really based on light and dark settings more what page is shown on

File: docs/src/helpers/logo-helper.js (hunk 2 above)
Comment (ojcurtis) [reply docs/src/helpers/logo-helper.js]: I think `lightVariant` might be confusing as it doesn't depend on if the site is using a dark theme/mode more it is for solid vs coloured backgrounds

Comment (georgegillams) [reply docs/src/helpers/logo-helper.js]: can we rename it `overImage` or something then, to capture that this is for use on top of images?

Comment (ojcurtis) [reply docs/src/helpers/logo-helper.js]: Done!

**Pull request merged**